gemrb/includes/ie_types.h

```diff
@@ -54,6 +54,7 @@ using ieDwordSigned = signed long int;
 // obviously we will define any hardcoded strrefs we use here
 enum class ieStrRef : ieDword {
 	INVALID = ieDword(-1),
+	NO_TEXT = 0, // valid strref that is simply <NO TEXT>
```

Comment:
that's a poor name, since that's the value only in bg2 — can give the wrong impression. I see no value in naming this.

Reply:
prefix it with `BG2_`? "conflicting" strafes aren't a problem, it's just an alias.

Reply:
Yeah, but there's no need for it, since there's nothing special about strref 0.

Reply:
it seems from this discussion there _is_ something special about strref 0. assuming the subtitles for all games return 0 in this case then strref 0 at least can mean `NO_SUBTITLE` for all games (regardless of whatever else it can mean). if only BG2 is returning 0 then we are already hacking by checking for it always 🤷 

of course I likely introduced a bug by checking using > because `-1` unsigned is _much_ larger than 0.

Reply:
If you insist on using a constant, then yeah, something like NOT_A_SUBTITLE would convey it, for that tiny particular scope and nowhere else.